python/lsst/meas/algorithms/ingestIndexReferenceTask.py

```diff
@@ -238,6 +314,32 @@ def _set_mags(self, record, row, key_map):
                 record.set(key_map[err_key+'_fluxSigma'],
                            fluxErrFromABMagErr(row[error_col_name], row[err_key]))
 
+    def _set_proper_motion(self, record, row, key_map):
+        """Set the proper motions from the input
+
+        The proper motions are read from the specified columns,
+        scaled appropriately, and installed in the appropriate
+        columns of the output.
+
+        Parameters
+        ----------
+        record : `lsst.afw.table.SourceRecord`
+            Source record to modify.
+        row : `dict`-like
+            Row from numpy table.
+        key_map : `dict` mapping `str` to `lsst.afw.table.Key`
+            Map of catalog keys.
+        """
+        if self.config.pm_ra_name is None:  # IngestIndexedReferenceConfig.validate ensures all or none
+            return
+        record.set(key_map["pmRa"], row[self.config.pm_ra_name]*self.config.pm_scale)
+        record.set(key_map["pmDec"], row[self.config.pm_dec_name]*self.config.pm_scale)
+        record.set(key_map["epoch"], self.epoch_poly(row[self.config.epoch_name]))
+        if self.config.pm_ra_err_name is None or self.config.pm_dec_err_name is None:
+            return
+        record.set(key_map["pmRaErr"], row[self.config.pm_ra_err_name]*self.config.pm_scale)
+        record.set(key_map["pmDecErr"], row[self.config.pm_dec_err_name]*self.config.pm_scale)
```

Comment:
I plan to use whatever has been approved in the RFC. If you want something different, then you need to RFC it.

Reply:
Ouch. In my opinion having an inconsistent naming convention for fields in the schema will be a disaster. I'd much rather hold off merging this and deal with it as a new RFC if necessary. Is `coord_ra_err` mandated in an RFC?

Reply:
RFC 271 doesn't say anything about the names for coord errors, so I'm picking `coordRaErr` and `coordDecErr` to match the proper motion error names.

The down side to any of these name choices is that we cannot save them as pair type (unlike `coord` which is a `Coord` field that internally has two scalar fiels `coord_ra` and `coord_dec`). However, the only obvious pair type is `Point` and the scalar fields would have to have suffixes `_x` and `_y` which is hardly ideal for RA and Dec.

Reply:
`afw::table::CovarianceMatrixKey` is intended to support storing any kind of error, including diagonal covariances.  It uses the old conventions now, but if you use it, it'll be updated to RFC-271 automatically when `CovarianceMatrixKey` itself is updated.

The names should be `coord_raErr`/`coord_decErr`, and that's what `CovarianceMatrixKey` will produce.

Reply:
(note edits in my last post - I was confusing RFC-271 with RFC-333)

Reply:
@TallJimbo thank you for the suggestion and for re-opening RFC-271 to match RFC-333! I will proceed as if your modification to RFC-271 is accepted, since I will be surprised if it proves controversial.